LanguageSkills/Controllers/HomeController.cs

```diff
@@ -3,19 +3,25 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using BusinessLogicLayer;
 using DataAccessLayer;
 using DataAccessLayer.DataBaseModels;
 
 namespace LanguageSkills.Controllers
 {
     public class HomeController : Controller
     {
-
+        private UnitOfWork _unitOfWork = new UnitOfWork();
 
         public void Index()
         {
 
-            Console.WriteLine("Hello World");
+            //Category category = new Category();
```

Comment:
Please do not leave commented pieces in code - instead please a) add comment (e.g. required for Debug only), or b) just delete it